posts/Davide/2016-10-10-hibernate-ogm-5-1-alpha-and-5-0-2-released.adoc

```diff
@@ -0,0 +1,119 @@
+= "The first hibernate OGM 5.1 Alpha and 5.0 maintenance release"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Good news!
+
+We released http://hibernate.org/ogm/[Hibernate OGM] 5.1 Alpha1 and 5.0.2 Final!
+
+== What's new?
+
+=== Hibernate OGM 5.0.2.Final
+Hibernate OGM 5.0.2.Final supports now http://www.mongodb.com[MongoDB] 3.2 and it's stll
+compatible backward compatible with Hibernate OGM 5.0.1.Final.
+
+If you need to upgrade from a previous version, you can find help on the
+https://developer.jboss.org/wiki/HibernateOGMMigrationNotes[migration notes].
+
+The 
+https://github.com/hibernate/hibernate-ogm/blob/5.0.2.Final/changelog.txt[5.0.2.Final changelog]
+contains all the details about the changes in this release
+
+=== Hibernate OGM 5.1.0.Alpha1
+Hibernate OGM 5.1.0.Alpha1 contains our first attempt to support https://neo4j.com[Neo4j] in
+remote mode. The user will have the option to choose between the better performance of the Bolt
+protocol or the familiarity of the HTTP interface.
```

Comment:
The guy from Neo4j wasn't really affirmative about Bolt having better performances.

Maybe just "between the new Bolt protocol..."